CHANGELOG.md

```diff
@@ -1,185 +1,183 @@
-[GitHub query result]: https://github.com/magento-research/pwa-studio/pulls?page=1&q=is%3Apr+is%3Amerged+base%3Arelease%2F2.0+review%3Aapproved+NOT+fix+in%3Atitle+NOT+fixed+in%3Atitle+NOT+chore+in%3Atitle+NOT+test+in%3Atitle&utf8=%E2%9C%93
+| Change type | Description                                                                 | PR        |
+| ----------- | --------------------------------------------------------------------------- | --------- |
+| Feature     | New SwatchTooltip component                                                 | [#956][]  |
+| Feature     | Focus search input on search icon button click                              | [#1019][] |
+| Update      | Improve the usability of the "Remove item" feature in the MiniCart          | [#882][]  |
+| Tests       | Add component unit tests                                                    | [#1027][] |
+| Bugfix      | Make Create Account and Sign In input style consistent                      | [#1169][] |
+| Bugfix      | Populate **Create Account** fields with correct values after guest checkout | [#1153][] |
+| Bugfix      | Fix expired guest cart errors                                               | [#1150][] |
+| Bugfix      | Fix header logo width style                                                 | [#1070][] |
+| Bugfix      | Disable adding to cart until product options are selected                   | [#1097][] |
+| Bugfix      | Disable update cart button until product options are selected               | [#1125][] |
+| Bugfix      | Update `div` tag with `Fragment`                                            | [#1103][] |
+| Bugfix      | Fix configurable media loading issue                                        | [#1094][] |
+| Bugfix      | Use placeholder in carousel while loading next image                        | [#1085][] |
+| Bugfix      | Add `/` to graphql validation endpoint                                      | [#1045][] |
+| Bugfix      | Fix `makeUrl` for Fastly                                                    | [#1039][] |
+| Bugfix      | Hide menu item from navigation if it is disabled in the Magento admin       | [#1022][] |
+| Bugfix      | Prevent adding to cart during rapid multi-clicking                          | [#910][]  |
+
+### Buildpack changes
+
+| Change type | Description                           | PR        |
+| ----------- | ------------------------------------- | --------- |
+| Update      | Update `workbox-webpack-plugin` to v4 | [#1102][] |
+
+### Misc project changes
+
+| Change type    | Description                            | PR                                         |
+| -------------- | -------------------------------------- | ------------------------------------------ |
+| Update         | Update eslint configuration version    | [#1088][]                                  |
+| Infrastructure | CICD and devops fixes and improvements | [#1132][], [#1155][], [#1087][], [#1043][] |
```

Comment:
Should be 'DevOps' and not 'devops'. We could also consider changing CICD to CI/CD but that's more of a personal opinion.